### PR TITLE
Fix classType for artifacts.

### PR DESCRIPTION
**Pull request by thesamprice**
Artifacts currently have a class type of 3 ( any class)
This assigns them the correct class type.

### CODE REVIEW

File: app/scripts/services/dimStoreService.factory.js [hunk 1]
```diff
@@ -540,6 +540,12 @@
           index = index + item.itemInstanceId;
         }
 
+        /* 0: titan, 1: hunter, 2: warlock, 3: any */
+        var ArtifactTypes = ["Titan Artifact","Hunter Artifact","Warlock Artifact"]
+        var ArtifactIndex = ArtifactTypes.indexOf(itemDef.itemTypeName);
+        if(ArtifactIndex != -1)
```
Comment (bhollis) [inline app/scripts/services/dimStoreService.factory.js]: Would you mind putting braces around the if condition, and a space after "if"?

File: app/scripts/services/dimStoreService.factory.js [hunk 2]
```diff
@@ -540,6 +540,12 @@
           index = index + item.itemInstanceId;
         }
 
+        /* 0: titan, 1: hunter, 2: warlock, 3: any */
+        var ArtifactTypes = ["Titan Artifact","Hunter Artifact","Warlock Artifact"]
```
Comment (bhollis) [inline app/scripts/services/dimStoreService.factory.js]: To fit with the existing code, please add a semicolon and add commas after each array element. You probably also want to name the variable `artifactTypes` to fit with other variable names.

**Comment by bhollis** [comment]
This looks good, thanks!

File: app/scripts/services/dimStoreService.factory.js (hunk 2 above)
Comment (thesamprice) [inline app/scripts/services/dimStoreService.factory.js]: Done I think.

On Wed, Oct 21, 2015 at 1:10 AM, Ben Hollis <notifications@github.com>
wrote:

> In app/scripts/services/dimStoreService.factory.js
> <https://github.com/DestinyItemManager/DIM/pull/315#discussion_r42583831>:
>
> > @@ -540,6 +540,12 @@
> >            index = index + item.itemInstanceId;
> >          }
> >
> > +        /* 0: titan, 1: hunter, 2: warlock, 3: any */
> > +        var ArtifactTypes = ["Titan Artifact","Hunter Artifact","Warlock Artifact"]
>
> To fit with the existing code, please add a semicolon and add commas after
> each array element. You probably also want to name the variable
> artifactTypes to fit with other variable names.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/315/files#r42583831>.
>



-- 
Thank you,

Sam Price
(707) 742-3726

**Pull request merged**